azure_functions_worker/bindings/meta.py

```diff
@@ -184,3 +231,42 @@ def to_outgoing_param_binding(binding: str, obj: typing.Any, *,
         return protos.ParameterBinding(
             name=out_name,
             data=rpc_val)
+
+
+def deferred_bindings_decode(binding: typing.Any,
+                             pb: protos.ParameterBinding, *,
+                             pytype: typing.Optional[type],
+                             datum: typing.Any,
+                             metadata: typing.Any):
+    # This cache holds deferred binding types (ie. BlobClient, ContainerClient)
+    # That have already been created, so that the worker can reuse the
+    # Previously created type without creating a new one.
+    global deferred_bindings_cache
+
+    # If cache is empty or key doesn't exist, deferred_binding_type is None
+    if deferred_bindings_cache.get((pb.name,
+                                    pytype,
+                                    datum.value.content), None) is not None:
+        return deferred_bindings_cache.get((pb.name,
+                                            pytype,
+                                            datum.value.content))
+    else:
+        deferred_binding_type = binding.decode(datum,
+                                               trigger_metadata=metadata,
+                                               pytype=pytype)
+        deferred_bindings_cache[(pb.name,
+                                 pytype,
+                                 datum.value.content)] = deferred_binding_type
+        return deferred_binding_type
+
+
+def check_deferred_bindings_enabled(param_anno: type,
+                                    deferred_bindings_enabled: bool) -> (bool,
+                                                                         bool):
+    # The first bool represents if deferred bindings is enabled at a fx level
```

Comment:
You have mostly all the functions and methods with the "#" coments. Can you please move these to ``` comments for official PyDoc comments